scrapinghub/utils.py

```diff
@@ -41,10 +42,10 @@ def parse_job_key(jobkey):
     if len(parts) != 3:
         raise ValueError("Job key should consist of projectid/spiderid/jobid")
     try:
-        parts = map(int, parts)
+        map(int, parts)
     except ValueError:
         raise ValueError("Job key parts should be integers")
-    return JobKey(*parts)
+    return JobKey(*map(str, parts))
```

Comment:
Ah, seem like this entire file is something that has never been merged to the master, so probably we are good here... Correct me if I'm wrong